.ci/kontinuum.json

```diff
@@ -1,4 +1,4 @@
 {
 "type":"library",
-"stages":["build"]
```

Comment:
AFAIK check is executed by build - so this is a duplicate then

Reply:
I've removed this step from the build. I thought you would want the double execution as the initial stages were `test` and `build` and, AFAIK, the `build` task executes the `test` task too.